WORKSPACE

```diff
@@ -88,3 +88,9 @@ git_repository(
     remote = "https://github.com/GoogleCloudPlatform/runtimes-common.git",
     commit = "4e9b3b57efb237ba09b4a319bea42bcdd4eb91e3",
 )
+
+git_repository(
```

Comment:
Ah, I was originally trying to use their archive.py library to crack open the .deb, but then ran into a wall when I found Python can't open .xz files easily.